src/nvim/eval.lua

```diff
@@ -333,7 +334,7 @@ M.funcs = {
 
     ]=],
     name = 'assert_equalfile',
-    params = {},
+    params = { { 'fname-one', 'string' }, { 'fname-two', 'string' } },
     returns = '0|1',
     signature = 'assert_equalfile({fname-one}, {fname-two})',
```

Comment:
Future idea: generate the signature docs from the `args` + `params`, to eliminate this `signature` field?

Reply:
Seems like a relatively simple change we can be iterated by gradually removing `signature` fields for functions with `params`.